### ISSUE TITLE
A11y_CDE - Mission Control and APIs_Revision_High Contrast: Keyboard focus is not visible on list items in High Contrast Black and White modes.

### ISSUE THREAD

**Issue by Bhaskar301101** [facebook/react#22228]
Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

Environment Details:
URL:  https://mission-control-ci.azurewebsites.net/​
OS Details:
Microsoft windows 10 enterprise ​and 11 
Version: 10.0.18362 Build 18362.356

Browser Details:​
Version 90.0.818.66 (Official build) (64-bit)
NOTE: Issue is observed through out the application wherever dropdown items present.

Repro Steps:​
1.Enable High Contrast mode and hit the URL : https://mission-control-ci.azurewebsites.net/ and log in with appropriate credentials.
2.Tab till "Insider Fast" link present under Audience header in table and press enter.​
3.Tab till Adoption Trend-Revision tab.
4.Tab to "Data Slices" and press enter.
5.Tab to list items and verify focus is visible on list items or not.

Actual Result:​
Keyboard focus is not visible on list items in High Contrast Black and White modes.
​
Expected Result:​
Keyboard focus should be visible on list items in High Contrast Black and White modes.
​
User Impact:​
Low vision users find difficulty if focus is not visible on few controls on page.
​
MAS Reference:​
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
<img width="576" alt="bug4" src="https://user-images.githubusercontent.com/89897257/131654631-f054b2b0-67e5-4a0b-973c-509185a23b95.png">

**Comment by eps1lon** [comment]
This repository is not affiliated with https://mission-control-ci.azurewebsites.net/%E2%80%8B
